kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -145,30 +144,25 @@
         // creates array of words in filter, removes empty strings
```

Comment:
This wasn't in the current PR, but there's no reason to assign `this.roleFilter` to a second locally scoped variable.  Just reference `this.roleFilter` to remove a layer of indirection.